src/monitor.tf

```diff
@@ -62,3 +62,32 @@ resource "azurerm_monitor_action_group" "slack" {
 
   tags = var.tags
 }
+
+resource "azurerm_monitor_diagnostic_setting" "app_gw" {
```

Comment:
would it be possible to move this code block inside the application gateway module and add a condition based on the secret name which contains the workspace id?

Reply:
what do you mean with " add a condition based on the secret name which contains the workspace id"? if I am not wrong, this app gw module is custom for cstar only, so what can be the other conditions?